addon/components/bs-navbar.js

```diff
@@ -131,23 +128,23 @@ export default class Navbar extends Component {
    * @default true
    * @public
    */
-  @defaultValue
-  fluid = true;
+  get fluid() {
+    return this.args.fluid ?? true;
+  }
 
   /**
    * BS5 only: Allows to override the container layout, see https://getbootstrap.com/docs/5.0/components/navbar/#containers.
    * Allowed values: `'sm'`, `'md'`, `'lg'`, `'xl'`, `'xxl'`, `'fluid'`, see https://getbootstrap.com/docs/5.0/layout/containers/.
-   * By default it is `.container-fluid`, or `.container` if the `@fluid` argument is set to false.
+   * By default, it is `.container-fluid`, or `.container` if the `@fluid` argument is set to false.
    *
    * @property container
    * @type string
    * @public
    */
 
-  @computed('fluid', 'container')
   get containerClass() {
-    if (this.container) {
-      return `container-${this.container}`;
+    if (this.args.container) {
+      return `container-${this.args.container}`;
     }
```

Comment:
For readability I tend to prefer destructing.

```suggestion
    const { container } = this.args;
    
    if (container) {
      return `container-${container}`;
    }
```